scripts/drtprod

```diff
@@ -331,6 +341,15 @@ EOF"
         # setup dns
         $0 dns drt-chaos create
         ;;
+      "workload-chaos")
+        roachprod create workload-large \
+          --clouds gce \
+          --nodes 1 \
+          --gce-machine-type n2-standard-4 \
```

Comment:
Let's put `n2-standard-8` since the chaos workload runner does more varied work, and that also matches what we currently have.

Reply:
Updated